game-adivinhacao.rb

```diff
@@ -0,0 +1,32 @@
+cond = 0 
+
+puts " Bem-vindo ao jogo de adivinhação "
+puts " Qual o seu nome? "
+nome = gets.to_s
+
+puts "\n\n\n\n\n\n"
+puts "Começaremos o jogo pra você, " + nome
```

Comment:
Também funciona e é mais utilizado:
```suggestion
puts "Começaremos o jogo pra você, #{nome}"
```